sources/install/package_misc.sh

```diff
@@ -199,6 +199,17 @@ function install_uploader() {
     add-to-list "uploader,https://github.com/Frozenka/uploader,Tool for quickly downloading files to a remote machine based on the target operating system"
 }
 
+function install_wesng() {
+    # CODE-CHECK-WHITELIST=add-aliases
+    colorecho "Installing wesng"
+    pipx install --system-site-packages git+https://github.com/bitsadmin/wesng
+    pipx inject wesng chardet # Not mendatory but stop the warning
```

Comment:
hmmm, weird to install first through `pipx git+url` then inject it in itself. We probably don't need to keep the `inject wesng`. (also please remove the comment or explain what warning the `chardet` injection fixes)

Reply:
I keep pipx install because it the recommended way to install it.

The error is this one : `WARNING:root:chardet module not installed. In case of encoding errors, install chardet using: pip3 install chardet`

The inject is to correct this warning. The app have no issue but can have with some charset.

But I will delete the line ;)

Reply:
forget my previous comment on the pipx inject
I thought you were injecting `wesng`, in `wesng`, which is not the case of course
Please add this line again, I was wrong asking you to removing it

Reply:
It's added again ;)